sroka/api/s3_connection/README.md

```diff
@@ -33,4 +33,31 @@ s3_download_data('s3://bucket/folder/2019_01_01/', output_file='./clicked.csv',
 
 s3_download_data('s3://bucket/folder/2019_01_01/part-111-111-111-111-111-111.csv', 
    output_file='./clicked_1.csv')
-```
\ No newline at end of file
+```
+
+
+### `s3_upload_data(data, bucket, path, sep=',')`
+
+
+#### Arguments
+
+* pandas DataFrame or numpy array `data` - object to be saved on s3  (obligatory)
+* sting `bucket` - name of an existing bucket on s3 that is file destination (obligatory)
+* string `path` - full file path within the bucket (obligatory)
+* string `sep` - what separator is used in (default: `,`)
+
+#### Returns
+
+* confirmation the file is saved
```

Comment:
I'm worried that this message could be misleading as the function doesn't return anything, it only prints the message. But maybe it's just my concern - I'm just pointing